tornjak-frontend/src/components/entry-create-json.tsx

```diff
@@ -178,7 +178,7 @@ class CreateEntryJson extends Component<CreateEntryJsonProp, CreateEntryJsonStat
                 // populate first entry
                 this.setSelectedEntriesIds(0, 0, parsedData.entries[0])
             } catch (e) {
-                displayError("Encountered a parse error.")
+                showToast({caption: "Encountered a parse error."})
```

Comment:
Can we show here what the parse error is? Does `e` contain anything useful here? Have you tried to run it?

Reply:
Likely, I'll take a look at this